app/templates/email/en_us/notifications/collaboration_approved.jinja2

```diff
@@ -1,525 +1,7 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" style="background:#f3f3f3!important"
+{% extends 'email/en_us/base.jinja2' %}
+{% block title %}Collaboration Approved, Flukebook{% endblock %}
```

Comment:
This is generally MUCH more readable and is a big improvement, Thanks for this. 
But there  are a few flukebook specific references, this is one of them and needs to go : )

Reply:
Not sure, I was goinng by what was in line 4

Reply:
By the way I'm not sure where `<title>Collaboration Approved, Flukebook</title>` actually is shown in the email...

![Screenshot 2022-02-24 at 16-32-59 You have a collaboration request from Houston on context_not_set - karen karen-chan com -](https://user-images.githubusercontent.com/153353/155566826-34ad9e38-cd93-45d4-83ba-45c0c33ff55f.png)

Reply:
Interesting, also not clear is where that flukebook image across the top comes from :)

Reply:
Yeah the title is in the html but is not displayed by the gmail web client.  The flukebook image is from the `header_image_url`.